.cicdtemplate/.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -16,10 +16,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          distribution: 'temurin'
+          java-version: '11'
```

Comment:
Seems like we missed one more file where we need to update Java 1.8:

`deploy_staging_and_production_to_firebase_app_distribution.yml`

Reply:
@Tuubz Please create another request to update the template and sample as well 👍

Reply:
Hmmm, are you suggesting we should create a new issue for this? 

But `deploy_staging_and_production_to_firebase_app_distribution.yml` is in the same folder as this file

Reply:
<img width="471" alt="Screen Shot 2022-11-17 at 10 31 45" src="https://user-images.githubusercontent.com/18277915/202348294-e9c37dd4-8a9f-4e60-a9d2-a2d13a1d1502.png">

Reply:
@Tuubz Oh, okay, it is a small think, lets me update it as well 👌